docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,98 @@
+
+**IntegrationTestScenario** +
+A parameter that the integration service uses to test application components.
+
+**Java build service** +
+A group of utilities that accelerates build times, locates dependencies in final images, and meets product security requirements by managing the construction of Java dependencies. 
+
+**lifecycle run** +
+The processes that automatically occur after a merge or pull request is resolved. These processes can include build tests, integration tests, and TaskRuns. 
+
+**managed workspace** +
+A workspace that mirrors some details of an already existing workspace. Supporting teams can create a managed workspace to grant limited permissions to development teams in the production environment. 
+
+**OSBS features** +
+The layered image build service, or OSBS (derived from OpenShift Build Service), is a tool for building layered images. With OSBS features, you can build base RHEL images, layered images, and the builder image itself.
+
+**persister** +
+A component that moves all of the relevant PipelineRun information, known as the Pipeline output, to an external storage that is outside of the cluster’s etcd database. The persister runs after the system completes the PipelineRun.
+
+**pipelines as code** +
+A practice that defines pipelines by using source code in Git. Pipelines as Code is also the name of link:https://pipelinesascode.com[a subsystem] that executes those pipelines.
+
+**pipeline results** +
+Systems that retain the history and details of builds. 
+
+**PipelineRun** +
+A collection of TaskRuns that are arranged in a specific order of execution. A valid pipeline is compliant with the customer’s enterprise contract.
+
+**pipeline service** +
+A managed service that securely runs Tekton pipelines. It also stores the metadata and pipeline logs that are related to an executed pipeline in a separate database.
+
+**product security test** +
+A test that assesses the security of build inputs and outputs with vulnerability scanning, code scanning, image testing, and result reporting. It also identifies outdated images, applications, and build pipelines. 
+
+**pruner** +
+A component that removes resources that are associated with the completed PipelineRuns. The system assigns resources, such as pods, to every PipelineRun. Without a pruner, these resources remain in the cluster indefinitely, even after the system completes the PipelineRun. 
+
+**release pipeline** +
+A pipeline that defines the process for validating snapshots against the enterprise contract. They also provide an array of release destinations. 
+
+**security testing** +
+A process that determines if images meet security quality standards.
+
+**signature and provenance** +
+Mechanisms that use Tekton chains to extract and store the metadata of the PipelineRuns, sign that metadata, and then store it in the image registry that is next to the component image.
```

Comment:
Is there a reason that these are grouped together? Signatures are cryptographic associations with specific identities. Provenance (i.e. SLSA provenance) is a description of the process that built the artifact.

Konflux uses Chains to generate them, but the signature/provenance don't use chains ... they are an output from Chains.

https://slsa.dev/attestation-model
https://slsa.dev/spec/v1.0/provenance
https://in-toto.readthedocs.io/en/latest/command-line-tools/in-toto-sign.html

Reply:
I'm not sure where that signature and provenance definition came from, but frankly it didn't seem great. I rewrote it, hopefully with better results.